iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/mapreduce/IcebergInputFormat.java

```diff
@@ -380,7 +380,8 @@ public void close() throws IOException {
         case PIG:
         case HIVE:
           // TODO implement value readers for Pig and Hive
-          throw new UnsupportedOperationException("Avro support not yet supported for Pig and Hive");
+          throw new UnsupportedOperationException("Vectorized execution is not yet supported for Iceberg avro " +
```

Comment:
We might want to separate out this to a different PR

Reply:
Sure!

Reply:
https://github.com/apache/hive/pull/2965